src/main/kotlin/org/rust/lang/core/grammar/rust.bnf

```diff
@@ -662,8 +663,7 @@ type_qual_path ::= << qualified_path view_path_prefix >>
 
 private types ::= type !'=' (',' type !'=')*
 
-private fn_anon_params ::= '(' anon_parameter anon_params_allow_variadic_tail ')'
-                         | '(' ')'
+bare_fn_params ::= '(' [ anon_parameter anon_params_allow_variadic_tail ] ')' { pin = 1 }
```

Comment:
So what do you suggest? I have no strong feelings about what's (`bare_fn_params` or `fn_type_params`) more appropriate